src/component/FileManager/Dialogs/Share/ShareSetting.tsx

```diff
@@ -150,7 +152,28 @@ const ShareSettingContent = ({ setting, file, editing, onSettingChange }: ShareS
             </ListItemSecondaryAction>
           </StyledListItemButton>
         </AccordionSummary>
-        <AccordionDetails>{t("application:modals.privateShareDes")}</AccordionDetails>
+        <AccordionDetails sx={{ display: "flex", flexDirection: "column", alignItems: "center" }}>
+          <Typography>{t("application:modals.privateShareDes")}</Typography>
+            {setting.is_private && (
```

Comment:
Let use a checkbox or Accordion to hide password related form.

For example: a checkbox `Custom share link password`, and show password textfield only when the checkbox is checked.